pinot-tools/src/main/java/org/apache/pinot/tools/perf/QueryRunner.java

```diff
@@ -157,6 +161,11 @@ public boolean execute()
       return false;
     }
 
+    SSLContext ctx = SSLContext.getInstance("TLS");
```

Comment:
Does this impact non tls situation? Shall we have a flag for it?

Reply:
yeah. I was thinking of that but got lazy... I may need to keep SSLContext outside the if flag {} to make it work. Let me do some research and add the flag.

Reply:
Done